### PR TITLE
add siphon-subset notebook

### PR DESCRIPTION
**Pull request by haileyajohnson**
- address #19

### CODE REVIEW

File: notebooks/dataAccess/siphon-Subset.ipynb [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+    "<h1>Siphon (subset)</h1>\n",
+    "<h3>Unidata AMS 2021 Student Conference</h3>\n",
+    "\n",
+    "<div style=\"clear:both\"></div>\n",
+    "</div>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "This notebook will demonstrate how to use Siphon to subset and download data using the NetcdfSubset service (NCSS). NCSS supports coordinate-based subsetting, i.e. selecting data by latitude, longitude, time, etc.\n",
+    "<div style=\"float:right; width:250 px\"><img src=\"../../instructors/images/siphon_subset_preview.png\" alt=\"plot of requestes subset of data\" style=\"height: 300px;\"></div>\n",
```
Comment (lesserwhirls) [inline notebooks/dataAccess/siphon-Subset.ipynb:23]: Looks like the image file ends in `.PNG`, which is breaking the link here, that ends in `.png`.

**Review by lesserwhirls — CHANGES_REQUESTED** [review]

File: notebooks/dataAccess/siphon-Subset.ipynb [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+   "outputs": [],
+   "source": [
+    "query = ncss.query()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will populate this query qith the same values as before, except instead of `latlon_point` we'll use `latlon_box`."
```
Comment (srcarter3) [inline notebooks/dataAccess/siphon-Subset.ipynb:351]: "populate this query **with**..."

File: notebooks/dataAccess/siphon-Subset.ipynb [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+    "temp = point_data.variables['Temperature_surface'][:] # get surface temperature data\n",
+    "time = point_data.variables['time'][:] # get time data\n",
+    "plt.plot(time, temp, 'k-'); # plot data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Subset data for a gridded region\n",
```
Comment (srcarter3) [inline notebooks/dataAccess/siphon-Subset.ipynb:327]: I might actually break these two subsections into the higher level objectives and have one for point data and one for gridded data, so they obviously show up in the outline.  But it's not necessary if you don't feel like it should be that way.

**Review by srcarter3 — CHANGES_REQUESTED** [review]
A few small spelling mistakes, and one organizational suggestion.

**Pull request merged**